src/main/scala/com/github/unchama/seichiassist/subsystems/gachapoint/domain/gachapoint/GachaPoint.scala

```diff
@@ -38,6 +38,8 @@ case class GachaPoint(exp: SeichiExpAmount) {
   }
 
   def add(point: GachaPoint): GachaPoint = GachaPoint(exp.add(point.exp))
+
+  def consume(point: GachaPoint): GachaPoint = GachaPoint(exp.consume(point.exp))
```

Comment:
```suggestion
  def subtract(point: GachaPoint): GachaPoint = GachaPoint(exp.consume(point.exp))
```
`add`関数の逆のことをやる関数なので、`consume`ではなく`subtract`のほうが良さそうです